src/targets/ref/target.cpp

```diff
@@ -42,17 +43,20 @@ std::string target::name() const { return "ref"; }
 
 std::vector<pass> target::get_passes(migraphx::context&, const compile_options&) const
 {
-    return {normalize_ops{},
-            eliminate_pad{},
-            dead_code_elimination{},
-            insert_pad{},
-            dead_code_elimination{},
-            rewrite_rnn{},
-            dead_code_elimination{},
-            auto_contiguous{},
-            dead_code_elimination{},
-            lowering{},
-            dead_code_elimination{}};
+    return {
+        normalize_ops{},
+        eliminate_pad{},
+        dead_code_elimination{},
+        eliminate_data_type{{migraphx::shape::fp8e4m3fnuz_type}, shape::float_type, {"quant_dot"}},
```

Comment:
Why wouldn't this be set to `all`?

Reply:
I didn't find it necessary for any other Ops

Reply:
I think its better to be granular here instead of heavy-handed. Atleast it's explicit what's being removed incase there's issues down the line.